app/src/main/java/com/github/multimatum_team/multimatum/util/PDFUtil.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.multimatum_team.multimatum.util
+
+import android.content.Intent
+import android.database.Cursor
+import android.net.Uri
+import android.provider.OpenableColumns
+
+
+object PDFUtil {
+    /**
+     * Intent for navigating to the files to select PDF
+     */
+    fun selectPdfIntent(callback: (Intent) -> Unit) {
```

Comment:
Why did you need to create this object instead of putting the function in the `AddDeadlineActivity.selectPDF` method?